propagators/opencensus/examples/README.md

```diff
@@ -0,0 +1,19 @@
+# OpenCensus binary propagation example
+
+The server uses OpenTelemetry with the opencensus binary propagation format.
+The client uses OpenCensus, which is hard-coded to use the opencensus binary
+propagation format. Since the client and server use the same propagation
+format, the ParentSpanID from the server spans should match the SpanID from
+the client spans, and both should share the same TraceID.
+
+### Usage
+
+First, start the opentelemetry server:
+```bash
+go run opentelemetry_server/server.go
+```
+
+In another shell, start the OpenCensus client:
+```bash
+go run opencensus_client/client.go
+```
```

Comment:
Can we add a sentence or two about what to expect from the example? Like expected STDOUT output?

Reply:
Done.